apis/r/tests/testthat/test-write-soma-resume.R

```diff
@@ -105,8 +107,8 @@ test_that("Resume-mode data frames", {
     )
   }
 
-  # Expect error when writing to existing array
-  expect_error(write_soma(co2, uri = uri, soma_parent = collection))
+  # Expect no error when writing to existing array as it will be skipped
+  expect_silent(write_soma(co2, uri = uri, soma_parent = collection))
 
```

Comment:
We do need to have some kind of catch here

Reply:
As discussed with @nguyenv we currently just skip.

Reply:
Hmm, is there a way we can catch the no-op and throw an error from tiledbsoma-r? I don't like the silent fail; if a user is trying to write to something that already exists, and hasn't explicitly said it's okay, we should throw some kind of message as otherwise the user will think "hey this worked"

Reply:
We could -- but I think the throw has to happen at the C++ level and propagate to R and Python.  And then be tested by both.

Reply:
Nevermind -- I was wrong. I do that on the C++ side of the R package.  I can easily error out. Trying that now.